src/App.tsx

```diff
@@ -1,18 +1,67 @@
-import React from 'react';
+import React, { useState } from 'react';
+
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
+import { ArrayNumber } from './Types/ArrayNumber';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
why did u disable this rule? consider removing unused vars

Reply:
this rule was disabled from the beggining after installing the npm. so I didn't touch it. 

and now when I enable it it gives me error and i can't fix it

Reply:
![image](https://github.com/mate-academy/react_pagination/assets/105914602/f1f7ac9c-886e-4fbf-a940-0814733cc6b7)
and when i put ';' as expected - still it doesn't work. so that is why i left it as it was.